blog/admin.py

```diff
@@ -0,0 +1,27 @@
+from django.contrib import admin
+from .models import Post, User, Commentary
+from django.contrib.auth.models import Group
+
+admin.site.register(Post)
+admin.site.register(User)
+admin.site.register(Commentary)
+admin.site.unregister(Group)
+
+
+class PostAdmin(admin.ModelAdmin):
+    search_fields = ("created_time", "owner", "title", "content")
+    list_filter = ("created_time", "owner", "title", "content")
```

Comment:
The fields 'title' and 'content' might not be the best choices for filtering. Consider using fields that are more likely to have a limited set of values, such as 'created_time' or 'owner'.